examples/querying-tables/querying_tables.bal

```diff
@@ -10,14 +10,26 @@ type Employee record {|
 public function main() {
     table<Employee> key(id) employees = table [
         {id: 1, firstName: "John", lastName: "Smith", salary: 100},
-        {id: 2, firstName: "Fred", lastName: "Bloggs", salary: 200}
+        {id: 2, firstName: "Jane", lastName: "Smith", salary: 150},
+        {id: 4, firstName: "Fred", lastName: "Bloggs", salary: 200},
+        {id: 7, firstName: "Bobby", lastName: "Clark", salary: 300},
+        {id: 9, firstName: "Cassie", lastName: "Smith", salary: 250}
     ];
 
-    // `from` clause iterates `employees` `table`.
-    // The contextually-expected type of the `query expression` is an `int[]`.
+    // The `from` clause iterates `employees` table.
+    // The query expression creates an integer `array` based on the contextually-expected type.
```

Comment:
This is slightly misleading. Gives the impression that it always gives an int array based on the contextually expected type. This should be phrased in a way that conveys what happens in this particular context because I don't think we can fit in the complete behaviour of a query expression when it comes to determining the type of the query expression. So in this case, there's a contextually expected type for the query expr, which is `int[]`, and therefore, it will create an `int[]` value upon its evaluation.

Reply:
Maybe something like "Since an integer array is contextually expected, the evaluation of the query expression will result in an integer array"?

Reply:
@MaryamZi thoughts?

Reply:
How about
```suggestion
    // Since the contextually-expected type for the query expression is `int[]` evaluation of the query expression will result in an integer array.
```